bin/crew

```diff
@@ -56,7 +57,7 @@ DOC = <<~DOCOPT
     crew reinstall [options] [-k|--keep] [-s|--source] [-S|--recursive-build] [-v|--verbose] <name> ...
     crew remove [options] [-v|--verbose] <name> ...
     crew search [options] [-v|--verbose] [<name> ...]
-    crew sysinfo [-v|--verbose] [options]
+    crew sysinfo [-v|--verbose]
```

Comment:
I'm not sure what `[options]` does but removing it doesn't seem to affect anything.